tpu-client/src/nonblocking/tpu_client.rs

```diff
@@ -325,6 +364,22 @@ where
         self.try_send_wire_transaction(wire_transaction).await
     }
 
+    /// Send a transaction to the current and upcoming leader TPUs according to fanout size
+    /// Returns a detached running task with the last error if all sends fail
+    pub async fn try_send_transaction_detached(
+        &self,
+        transaction: &Transaction,
+    ) -> JoinHandle<TransportResult<()>> {
+        let leaders = self
+            .leader_tpu_service
+            .leader_tpu_sockets(self.fanout_slots);
+        let wire_transaction = serialize(transaction).expect("serialization should succeed");
+        let connection_cache = self.connection_cache.clone();
+        task::spawn(async move {
+            try_send_wire_transaction(wire_transaction, &leaders, &connection_cache).await
```

Comment:
Have you tried just send the Arc<ConnectionCache> instead of the reference to try_send_wire_transaction so that we may avoid the 'static'?

Reply:
As mentioned at https://github.com/solana-labs/solana/pull/32844#discussion_r1296276718, it's around the lifetime of the *type* and not of the *instance*, so an Arc doesn't do anything unfortunately

Reply:
Got it. Thanks